service/src/test/java/bio/terra/cbas/controllers/TestPublicApiController.java

```diff
@@ -0,0 +1,56 @@
+package bio.terra.cbas.controllers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import bio.terra.cbas.model.CapabilitiesResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.io.Resource;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ActiveProfiles("test")
+class TestPublicApiController {
+
+  @Autowired MockMvc mockMvc;
+  @Autowired ObjectMapper objectMapper;
+
+  @Value("classpath:capabilities.json")
+  Resource capabilitiesResource;
+
+  private static final String CAPABILITIES_API = "/capabilities/v1";
+
+  @Test
+  void successfullyReturnCapabilities() throws Exception {
+    MvcResult mvcResult =
+        mockMvc.perform(get(CAPABILITIES_API)).andExpect(status().isOk()).andReturn();
+
+    CapabilitiesResponse parsedResponse =
+        objectMapper.readValue(
+            mvcResult.getResponse().getContentAsString(), CapabilitiesResponse.class);
+
+    assertThat(parsedResponse).isNotNull();
+
+    assertEquals(100, parsedResponse.get("submission.limits.maxWorkflows"));
```

Comment:
Something to consider - do we want to be using exact values from the actual `capabilities.json` in these tests? Could we have something maybe split in two halves, like 

- Test that the real file is readable by the service
- Test that we can interpret a mock file correctly and get the right values from it

Reply:
That is a good point. I do have a test case for this but I see your point about the downsides for directly testing the actual json file for exact values. I will make that change.
> Test that the real file is readable by the service

Reply:
I actually simply updated this test case to check that the response returned from API is parseable into CapabilitiesResponse model and verify it is not empty. I think this should cover most of the cases. If there is a need to have a test case read from mock file then we can add that later.